examples/js/loaders/OBJLoader.js

```diff
@@ -658,13 +689,25 @@ THREE.OBJLoader = ( function () {
 							materialLine.copy( material );
 							material = materialLine;
 
+						} else if ( isPoints && material && ! ( material instanceof THREE.PointsMaterial ) ) {
+
+							var materialPoints = new THREE.PointsMaterial( { size: 10, sizeAttenuation: false } );
+							materialLine.copy( material );
+							material = materialPoints;
+
 						}
 
 					}
 
 					if ( ! material ) {
 
-						material = ( ! isLine ? new THREE.MeshPhongMaterial() : new THREE.LineBasicMaterial() );
+						if ( isLine ) {
```

Comment:
Please ensure to follow the code style. You can check your code with https://zz85.github.io/mrdoobapproves/ 
This section should be look like this:
```javascript
if ( isLine ) {

   material = new THREE.LineBasicMaterial();

} else if ( isPoints ) {

   material = new THREE.PointsMaterial( { size: 1, sizeAttenuation: false } );

} else {

   material = new THREE.MeshPhongMaterial();

}